docker/koordlet.dockerfile

```diff
@@ -7,6 +7,7 @@ ENV VERSION $VERSION
 ENV GOOS linux
 ENV GOARCH $TARGETARCH
 
+RUN sed -i 's#deb.debian.org#ftp.cn.debian.org#' /etc/apt/sources.list
```

Comment:
What do you think?

Reply:
refer https://www.debian.org/mirror/list

Reply:
This modification is a localized improvement and may not help the GitHub workflow.

Reply:
OK, removed.